fw/cfg.h

```diff
@@ -191,6 +191,21 @@ typedef struct {
 	     (idx) < (e)->val_n;		\
 	     (v) = (e)->vals[++(idx)])
 
+#define TFW_CFG_CHECK_NO_ATTRS(spec, entry)				\
```

Comment:
I think we should can implement `tfw_cfg_check_attr_n` function with same as in `tfw_cfg_check_val_n` error message.
```
T_ERR_NL("invalid number of attributes; expected: %d, got: %zu\n",
			 attr_n, ce->attr_n);
```